docker-compose.test.yml

```diff
@@ -0,0 +1,19 @@
+version: '2'
+services:
+  test:
+    build: .
+    command: bash -c "sleep 20 && npm test"
+    depends_on:
+      - db
+    environment:
+      - DEBUG=
+      - MONGODB_URL=mongodb://db:27017/test
+      - OPLOG_MONGODB_URL=mongodb://db:27017/local
+  db:
+    image: mongo:2.4.14
+    command: mongod --replSet rs0 --oplogSize 20 --noprealloc  --smallfiles --nojournal
+  initReplicaset:
+    image: mongo:2.4.14
+    command: bash -c "sleep 3 && mongo admin --host db:27017 --eval 'printjson(rs.initiate());'"
+    links:
```

Comment:
This is not necessary in docker-compose v2, they are all part of the same network